custodia/cli/__init__.py

```diff
@@ -202,11 +204,14 @@ def main():
     if args.certfile:
         args.client_conn.set_client_cert(args.certfile, args.keyfile)
         args.client_conn.headers['CUSTODIA_CERT_AUTH'] = 'true'
-
     try:
         result = args.func(args)
     except RequestsHTTPError as e:
         return main_parser.exit(1, str(e))
+    except ConnHTTPError:
+        connection_error_msg = "Custodia is down or not responding in the " \
+                               "socket: %s" % args.server.path
```

Comment:
Please change the message to:
"Unable to connect to the server via {}".format(args.server.path)